cmake/Modules/GrPython.cmake

```diff
@@ -168,6 +168,7 @@ function(GR_PYTHON_INSTALL)
     install(
       FILES ${GR_PYTHON_INSTALL_FILES}
       DESTINATION ${GR_PYTHON_INSTALL_DESTINATION}
+      ${GR_PYTHON_INSTALL_UNPARSED_ARGUMENTS}
```

Comment:
Do the cmake changes need to be in the same PR?

Reply:
not really, but they were required in the previous iteration of it. Can extract it to a separate one.

Reply:
That would be good. I patched my system for a test, expecting a python-only, quick install. Got a cmake run.

Reply:
@willcode #4717